client/app/main/main.component.spec.js

```diff
@@ -76,7 +76,7 @@ describe('Component: MainComponent', function() {
       //this test catches if something goes wrong with that formatting
       mainComponent.$onInit();
       $httpBackend.flush();
-      expect(mainComponent.lastBuilt).to.equal("Wednesday, November 8, 2017 8:48 AM");
+      expect(mainComponent.lastBuilt).to.match(/[A-Za-z]*, [A-Za-z]* \d{1,2}, \d{4} \d{1,2}:\d\d [AP]M/);
```

Comment:
Instead of \d\d I'd just do \d{2} for consistency.

Reply:
Also, I'd use + intead of * since the comma wasn't made optional. That way we're guaranteed at least something in the day of week and month parts.